src/main/java/org/sitenv/vocabularies/loader/code/CptLoader.java

```diff
@@ -0,0 +1,93 @@
+    private String oid;
+
+    public CptLoader() {
+        this.oid = CodeSystemOIDs.CPT4.codesystemOID();
+    }
+
+    public long load(List<File> filesToLoad, DataSource ds) {
+    	long n = 0;
+    	JdbcTemplate t = new JdbcTemplate(ds);
+        BufferedReader br = null;
+        FileReader fileReader = null;
+        String insertQueryPrefix = codeTableInsertSQLPrefix;
+        try {
+            StrBuilder insertQueryBuilder = new StrBuilder(codeTableInsertSQLPrefix);
+            int totalCount = 0, pendingCount = 0;
+
+            for (File file : filesToLoad) {
+                if (file.isFile() && !file.isHidden()) {
+                    logger.debug("Loading CPT File: " + file.getName());
+                    String codeSystem = file.getParentFile().getName();
+                    fileReader = new FileReader(file);
+                    br = new BufferedReader(fileReader);
+                    String line;
+                    while ((line = br.readLine()) != null) {
+                        if (!line.isEmpty()) {
+                            String code = line.substring(0, 5);
+                            String displayName = line.substring(line.indexOf(" "));
+                            
+                            
+                            
+//                            buildCodeInsertQueryString(insertQueryBuilder, code, displayName, codeSystem, oid, CODES_IN_THIS_SYSTEM_ARE_ALWAYS_ACTIVE);
+//                            if ((++totalCount % BATCH_SIZE) == 0) {
+//                                insertCode(insertQueryBuilder.toString(), connection);
+//                                insertQueryBuilder.clear();
+//                                insertQueryBuilder.append(codeTableInsertSQLPrefix);
+//                                pendingCount = 0;
+//                            }
+                            
+                            //"insert into CODES (CODE, DISPLAYNAME, CODESYSTEM, CODESYSTEMOID) values (?,?,?,?)";
+                        	n++;
+                            t.update(insertQueryPrefix,code.trim().toUpperCase(),displayName.trim().toUpperCase(),file.getParentFile().getName(),oid);
+                            logger.trace("Inserted CPT Code : " + code + " and displayName : " + displayName);
+                        }
+                    }
+                }
+            }
+//            if (pendingCount > 0) {
+//                insertCode(insertQueryBuilder.toString(), connection);
+//            }
+        } catch (IOException e) {
+            logger.error(e);
+//        } catch (SQLException e) {
+//            e.printStackTrace();
+        } finally {
+            if (br != null) {
+                try {
+                    fileReader.close();
+                    br.close();
+                } catch (IOException e) {
+                    logger.error(e);
+                }
+            }
+        }
+        logger.debug("Number of CPT Codes processed : " + n);
+        return n;
+    }
+}
```

Comment:
If you have JUnit tests already created for the CptLoader, please add it to this Pull Request so we have it.  If you don't have it, don't worry about it.
When you delete the CPT folder and its contents as Margo requested, if we have the JUnit tests please add @Ignore to them so the tests won't fail the build.

Reply:
I realized we keep the vocabs in ais-validator repo instead of the OSS repos.  So JUnit test file, if any, should be checked into the ais-validator project.